v03_pipeline/lib/tasks/delete_project_family_tables.py

```diff
@@ -21,21 +23,24 @@ def complete(self) -> bool:
         )
 
     def run(self):
-        project_table_task: luigi.Target = yield HailTableTask(
-            project_table_path(
+        for sample_type in SampleType:
+            project_ht_path = project_table_path(
                 self.reference_genome,
                 self.dataset_type,
+                sample_type,
                 self.project_guid,
-            ),
-        )
-        project_ht = hl.read_table(project_table_task.path)
-        family_guids = hl.eval(project_ht.globals.family_guids)
-        for family_guid in family_guids:
-            self.dynamic_delete_family_table_tasks.add(
-                DeleteFamilyTableTask(
-                    reference_genome=self.reference_genome,
-                    dataset_type=self.dataset_type,
-                    family_guid=family_guid,
-                ),
             )
+            if hfs.exists(project_ht_path):
```

Comment:
@bpblanken I think what was happening before was that `HailTableTask` was actually never `complete()` if the project table didn't exist and that's what was causing the infinite loop?

yielding inside the sample_type loop works now if the project table exists.

Reply:
🤔 definitely strange behavior!  I think we do actually want this task to fail though if the project able isn't there right?

Reply:
I don't think so, all of the existing seqr projects will have project tables in only one sample_type directory.

Reply:
TODO: raise error if there is not at least 1 project table! else the task will never complete!

Reply:
^ done